webclient/js/elems/raflow/people.js

```diff
@@ -240,60 +256,15 @@ window.loadRAPeopleForm = function () {
-                {name: 'PriorLandLordPhoneNo',      type: 'text'},                          // Prior landlord's phone number
-                {name: 'PriorLengthOfResidency',    type: 'int'},                           // Length of residency at Prior address
-                {name: 'PriorReasonForMoving',      type: 'text'},                          // Reason of moving from Prior address
-                {name: 'Evicted',                   type: 'checkbox',   required: false },  // have you ever been Evicted
-                {name: 'EvictedDes',                type: 'text',       required: false },
-                {name: 'Convicted',                 type: 'checkbox',   required: false },  // have you ever been Arrested or convicted of a crime
-                {name: 'ConvictedDes',              type: 'text',       required: false },
-                {name: 'Bankruptcy',                type: 'checkbox',   required: false },  // have you ever been Declared Bankruptcy
-                {name: 'BankruptcyDes',             type: 'text',       required: false },
-                {name: 'Employer',                  type: 'text',       required: false },
-                {name: 'WorkPhone',                 type: 'text',       required: false },
-                {name: 'Address',                   type: 'text',       required: false },
-                {name: 'Address2',                  type: 'text',       required: false },
-                {name: 'City',                      type: 'text',       required: false },
-                {name: 'State',                     type: 'list',       required: false,    options: {items: app.usStateAbbr}, },
-                {name: 'PostalCode',                type: 'text',       required: false },
-                {name: 'Country',                   type: 'text',       required: false },
-                {name: 'Position',                  type: 'text',       required: false },
-                {name: 'GrossIncome',               type: 'money',      required: false },
-                {name: 'Comment',                   type: 'text'},                          // In an effort to accommodate you, please advise us of any special needs
-                {name: 'EmergencyContactName',      type: 'text',       required: false },  // Name of emergency contact
-                {name: 'EmergencyContactPhone',     type: 'text',       required: false },  // Phone number of emergency contact
-                {name: 'EmergencyContactAddress',   type: 'text',       required: false }   // Address of emergency contact
-            ],
+            fields: app.transactantFields,
+            tabs: app.transactantTabs,
             actions: {
                 save: function () {
                     var form = this,
                         TMPTCID = form.record.TMPTCID;
 
                     var errors = form.validate();
+                    console.log(errors);
```

Comment:
later comment this line.

Reply:
I meant in database migration too? You should search for keyword CompanyName or Employer throughout source tree and replace it everywhere!